group_vars/testnet.yml

```diff
@@ -2,7 +2,7 @@
 # maintains versions and can overwrite all.yml
 aztec:
   validator:
-    version: 0.87.6
+    version: 0.87.7
```

Comment:
What about https://www.notion.so/encapsulate/0-87-7-20b14f565a50804ea43df78b51111957?source=copy_link#20b14f565a5080d0a816fd36b49f0955 , do we not need it ?

Reply:
If you check the previous link, I think we can env variable AUTO_UPDATE=config to have the same effect maybe ?

Reply:
AUTO_UPDATE=config-and-version